app/components/point-input.jsx

```diff
@@ -0,0 +1,190 @@
+// https://github.com/EnCiv/civil-pursuit/issues/44
+
+'use strict'
+import React, { useEffect, useRef, useState } from 'react';
+import { createUseStyles } from 'react-jss';
+import cx from 'classnames';
+import autosize from 'autosize';
+
+function PointInput(props) {
+    const { style={}, className="", maxWordCount = 30, maxCharCount = 100, defaultValue = {description: "", subject: ""}, onDone = ()=>{}, ...otherProps} = props
+    const classes = useStyles()
+    const [subject, setSubject] = useState(defaultValue?.subject ?? "")
+    const [description, setDescription] = useState(defaultValue?.description ?? "")
+    const [descWordCount, setDescWordCount] = useState(getDescWordCount(description))
+    const [subjCharCount, setSubjCharCount] = useState(getSubjCharCount(subject))
+    const textareaRef = useRef(null);
+
+    useEffect(() => {
+        autosize(textareaRef.current)
+
+        return () => {
+            autosize.destroy(textareaRef.current)
+        }
+    }, [])
+
+    function getDescWordCount(inputText) {
+        if (!inputText) return 0
+        return inputText.trim().split(/\s+/).length;
+    }
+
+    function getSubjCharCount(inputText) {
+        return inputText.length
+    }
+
+
+    const isDescValid = inputText => {
+        const wordCount = getDescWordCount(inputText)
+        return wordCount > 0 && wordCount <= maxWordCount
+    }
+
+    const isSubjValid = inputText => {
+        const charCount = getSubjCharCount(inputText)
+        return charCount > 0 && charCount <= maxCharCount
+    }
+
+    const handleSubjectChange = (value) => {
+        setSubject(value)
+        setSubjCharCount(getSubjCharCount(value))
+    }
+
+    const handleDescriptionChange = (value) => {
+        setDescription(value)
+        setDescWordCount(getDescWordCount(value))
+    }
+
+    const handleOnBlur = () => {
+        onDone((isSubjValid(subject) && isDescValid(description)), ({ subject, description }))
```

Comment:
this should be:
```
onDone({valid: isSubjValid(subject) && isDescValid(description)). value: { subject, description }}))
```
The pattern is that all components call onDone with an object {value: ..., valid: ...}   meaning an object that has the property names "valid" and "value".  English isn't can be confusing.

Reply:
got it thanks, made the change!